website/.prettierrc.json

```diff
@@ -4,6 +4,6 @@
   "trailingComma": "es5",
   "bracketSpacing": false,
   "arrowParens": "always",
-  "proseWrap": "always",
+  "proseWrap": "never",
```

Comment:
The third option is `"preserve"` which means you put a newline it gets to stay. I don't love that, so I've gone with always.